<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Исходные данные</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>271</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Исходный файл</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fileName">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>110</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить файл</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>321</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск идентификатора</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ident">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>230</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>280</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Сброс</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>290</y>
      <width>151</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Всего поиск:</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>301</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>511</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>831</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>290</y>
      <width>51</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>раз</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_count">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>290</y>
      <width>108</width>
      <height>33</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Количество выполнявшихся операций поиска идентификаторов&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>340</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти все</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>400</y>
      <width>201</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Рехэширование</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>400</y>
      <width>211</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Бинарное дерево</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>440</y>
      <width>401</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(186, 189, 182);
background-color: rgb(238, 238, 236);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_5">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>440</y>
      <width>401</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(186, 189, 182);
background-color: rgb(238, 238, 236);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>570</y>
      <width>271</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>В среднем сравнений:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>530</y>
      <width>221</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Всего сравнений: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>450</y>
      <width>371</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Идентификатор </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>490</y>
      <width>171</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Сравнений:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>650</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход из программы</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_compare_1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>490</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_compareAll_1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>530</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_compareAvr_1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>570</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_compareAll_2">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>530</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_compareAvr_2">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>570</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>490</y>
      <width>171</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Сравнений:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>530</y>
      <width>221</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Всего сравнений: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_compare_2">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>490</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>450</y>
      <width>371</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Идентификатор </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>570</y>
      <width>271</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>В среднем сравнений:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>281</width>
      <height>581</height>
     </rect>
    </property>
   </widget>
   <zorder>graphicsView_3</zorder>
   <zorder>graphicsView_2</zorder>
   <zorder>graphicsView</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>lineEdit_fileName</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>label_3</zorder>
   <zorder>lineEdit_ident</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_count</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>graphicsView_4</zorder>
   <zorder>graphicsView_5</zorder>
   <zorder>label_12</zorder>
   <zorder>label_13</zorder>
   <zorder>label_14</zorder>
   <zorder>label_15</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>label_compare_1</zorder>
   <zorder>label_compareAll_1</zorder>
   <zorder>label_compareAvr_1</zorder>
   <zorder>label_compareAll_2</zorder>
   <zorder>label_compareAvr_2</zorder>
   <zorder>label_21</zorder>
   <zorder>label_22</zorder>
   <zorder>label_compare_2</zorder>
   <zorder>label_24</zorder>
   <zorder>label_25</zorder>
   <zorder>listWidget</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
